admin/base/settings/defaults.py

```diff
@@ -25,18 +25,34 @@
     '.osf.io'
 ]
 
+# Email settings. Account created for testing. Password shouldn't be hardcoded
```

Comment:
Exactly. It does all the template processing for the email and then instead of sending it via SMTP it prints the rendered email template to the console.